test/controllers/users.test.js

```diff
@@ -188,3 +193,60 @@ describe('usersController.listAllUsers', () => {
       .expect(401)
       .end(done));
 });
+
+describe('usersController.listUserAlbums', () => {
+  it('lists user albums for a non admin user', () => {
+    createAndSignInUser().then(({ createdUser, token }) =>
+      factory.createMany('userAlbumAssoc', 3, { userId: createdUser.id }).then(userAlbums => {
+        const uri = `/users/${createdUser.id}/albums`;
+        return request
+          .get(uri)
+          .set('Authorization', token)
+          .expect(200)
+          .then(response => {
+            expect(response.body).toHaveProperty('user_albums');
+            const responseUserAlbums = response.body.user_albums;
+            expect(responseUserAlbums.length).toBe(userAlbums.length);
+            return responseUserAlbums.forEach(responseUserAlbum => {
+              expect(responseUserAlbum).toHaveProperty('title');
+              expect(responseUserAlbum).toHaveProperty('created_at');
+              return expect(responseUserAlbum).toHaveProperty('updated_at');
+            });
+          });
+      })
+    );
+  });
+
+  it("lists another user's albums for an admin user", () =>
+    createAndSignInUser({ isAdmin: true }).then(({ token }) =>
+      factory.create('woloxUser').then(otherUser =>
+        factory.createMany('userAlbumAssoc', 3, { userId: otherUser.id }).then(userAlbums => {
+          const uri = `/users/${otherUser.id}/albums`;
+          return request
+            .get(uri)
+            .set('Authorization', token)
+            .expect(200)
+            .then(response => {
+              expect(response.body).toHaveProperty('user_albums');
+              const responseUserAlbums = response.body.user_albums;
+              expect(responseUserAlbums.length).toBe(userAlbums.length);
+              return responseUserAlbums.forEach(responseUserAlbum => {
+                expect(responseUserAlbum).toHaveProperty('title');
+                expect(responseUserAlbum).toHaveProperty('created_at');
+                return expect(responseUserAlbum).toHaveProperty('updated_at');
```

Comment:
idem

Reply:
Forgot that it wasn't necessary, removed in latest commit